globals/global_settings.gd

```diff
@@ -31,14 +32,15 @@ const user_settings_file = "user://settings.json"
 
 var user_settings = {
 	GameSound: true,
-	HideEnglishCardText: false,
+	HideEnglishCardText: true,
```

Comment:
If we're messing with this setting, I'd almost prefer to change it to 2 settings:
Something like "ShowCardTextOnCard" and "HideCardTextForProxies".
With the defaults being False (since you're adding this new thing) and True.
Toggling ShowCardTextOnCard would force the card text to always show on the card.
If HideCardTextForProxies is true, then it shows up even if there's a proxy card, if it's false, it only shows up if there is no proxy card.
UseEnProxies should probably also be renamed to like UseProxies or something.

Thoughts?

Reply:
I'm a bit confused with the settings you're proposing. Which card texts are we talking about here? Let's use these terms for clarity: **``overlay``** for the generated text that is within the card; **``panel``** for this feature.

Reply:
In that case, I guess it makes more sense with these names:
ShowCardOverlayText
HideCardOverlayTextForProxies

ShowCardOverlayText = true - the overlay always shows up.  False - it never shows up.
HideCardOverlayTextForProxies = true - hides the overlay text for proxy cards even if ShowCardOverlayText=true

The panel feature will show up if ShowCardOverlayText is false, which we can set as the default.

Does that make sense?

Reply:
> HideCardOverlayTextForProxies

I don't see a reason for this and it just overcomplicates things. What happens if ``hide_for_proxies`` is ``false`` and ``show_overlay`` is ``true``. That would make it so that the overlay will cover the card even if the image is fine. That would be bad UX.

I feel that the current settings we have works fine. It only shows the ``overlay/panel`` for cards we have no proxies for that particular language. We'd maybe just have to rename the setting to something better, maybe even add a tooltip.

Actually, I'm open to ditching adding the translations and use the generated text instead. That way, we can remove the ``overlay`` and just have the ``panel`` for the presentation.

Reply:
show overlay true and hide for proxies false is actually useful for debugging and checking how the strings show up on specific cards.
Also the overlay text is somewhat readable on the board compared to the proxy cards which are completely unreadable without zoom. I could see someone wanting to have them up all the time.

I'm definitely for just having one version of the strings, preferably the generated text.

Reply:
```
- Card Info [ ]
   -- Overlay [ ]
   -- Panel [ ]
```
I guess this would be the best setup?

I'd like to include the translations for effects without names for the panels though like collabs and they could be named appropriately like ``collab_id`` and such.

Reply:
That's fine